src/routes/expenses.route.js

```diff
@@ -0,0 +1,12 @@
+const express = require('express');
+const { expensesController } = require('../controllers/expenses.controller');
+
+const expensesRouter = express.Router();
+
+expensesRouter.get('/', expensesController.getAllExpenses);
+expensesRouter.post('/', expensesController.createExpense);
```

Comment:
```suggestion
expensesRouter.post('/', yourMiddlewareToCheckAndSanitizeTheRequest, expensesController.createExpense);
```
Middlewares should be placed here